tutorials/proxmox-softraid/index.mdx

```diff
@@ -66,15 +56,15 @@ The tool provides a REST-API as well as a graphical web interface to manage the
 
   <Lightbox src="scaleway-proxmox_installer-06.webp" width="550" alt="" />
 
-9. Configure the **Management Network**. This will be the network interface on which the Proxmox webinterface and APIs are available. It is recommended to configure it on the primary email address of your Scaleway Dedibox.
+9. Configure the **Management Network**. This will be the network interface on which the Proxmox webinterface and APIs are available. It is recommended to configure it on the primary email address of your Dedibox.
```

Comment:
```suggestion
9. Configure the **Management Network**. This will be the network interface on which the Proxmox web interface and APIs are available. We recommend you configure it on the primary email address of your Dedibox.
```